scripts/data/lpc43xx/csv2yaml.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
did you test this on py3? this file hasn't been touched in ~forever, and isn't guaranteed to be python3 safe